.github/workflows/ci.yml

```diff
@@ -25,7 +25,11 @@ jobs:
       matrix:
         jdk: [8, 11]
     env:
+      MVN_COMMON_OPTIONS: -U -B --no-transfer-progress
       MVN_CONNECTION_OPTIONS: -Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false -Dmaven.wagon.httpconnectionManager.ttlSeconds=120
+      FLINK_URL: https://s3.amazonaws.com/flink-nightly/flink-1.15-SNAPSHOT-bin-scala_2.12.tgz
```

Comment:
Why are we pointing at Flink 1.15-SNAPSHOT?

Reply:
Will use official Flink 1.15.2 at https://dlcdn.apache.org/flink/flink-1.15.2/flink-1.15.2-bin-scala_2.12.tgz